### PR TITLE
Rights access to the folders of the project

### PR DESCRIPTION
**Pull request by foxmask**
| Q             | A
| ------------- | ---
| Documentation | yes
| Translation   | yes
| Fixed tickets | #1983 
| License       | MIT

### CODE REVIEW

File: docs/en/user/installation.rst [hunk 1]
```diff
@@ -154,3 +154,32 @@ After reloading or restarting nginx, you should now be able to access wallabag a
 .. note::
 
     When you want to import large file into wallabag, you need to add this line in your nginx configuration ``client_max_body_size XM; # allows file uploads up to X megabytes``.
+
+
+
```
Comment (nicosomb) [inline docs/en/user/installation.rst]: Extra lines, you can remove them.

File: docs/en/user/installation.rst [hunk 2]
```diff
@@ -154,3 +154,32 @@ After reloading or restarting nginx, you should now be able to access wallabag a
...
+Test Environment
+~~~~~~~~~~~~~~~~
+
+When we just want to test Wallabag, we just run the command ``php bin/console server:run --env=prod`` to start our Wallabag instance and everything will go smoothly because the user who started the project can access to the current folder naturally, without any problem.
+
+
+Production Environment
+~~~~~~~~~~~~~~~~~~~~~~
+
+As soon as we use Apache or Nginx to access to our Wallabag instance, and not from the command  ``php bin/console server:run --env=prod`` to start it, we should take care to grant the good rights on the good folders to keep safe all the folders of the project.
```
Comment (nicosomb) [inline docs/en/user/installation.rst]: Can you replace Wallabag with wallabag please ? ;-)

File: docs/en/user/installation.rst [hunk 3]
```diff
@@ -154,3 +154,32 @@ After reloading or restarting nginx, you should now be able to access wallabag a
...
+
+To do so, the folder name, known as ``DocumentRoot`` (for apache) or ``root`` (for Nginx), has to be absolutely accessible by the Apache/Nginx user. Its name is generally ``www-data``, ``apache`` or ``nobody`` (depending on linux system used).
+
+So the folder ``/var/www/wallabag/web`` has to be accessible by this last one. But this could be not enough if we just care about this folder, because we could meet a blank page or get an error 500 when trying to access to the home page of the project.
```
Comment (nicosomb) [inline docs/en/user/installation.rst]: home page > homepage

File: docs/fr/user/installation.rst [hunk 4]
```diff
@@ -148,6 +148,34 @@ En imaginant que vous vouliez installer wallabag dans le dossier /var/www/wallab
...
+
+
+Droits d'accès aux dossiers du projet
+-------------------------------------
+
+
+Environnement de test
+~~~~~~~~~~~~~~~~~~~~~
+
+Quand nous souhaitons juste tester Wallabag, nous lançons simplement la commande  ``php bin/console server:run --env=prod`` pour démarrer l'instance Wallabag et tout se passe correctement car l'utilisateur qui a démarré le projet à accès naturellement au repertoire courant, tout va bien.
```
Comment (nicosomb) [inline docs/fr/user/installation.rst]: à accès > a accès

File: docs/fr/user/installation.rst [hunk 5]
```diff
@@ -148,6 +148,34 @@ En imaginant que vous vouliez installer wallabag dans le dossier /var/www/wallab
...
+
+
+Environnement de production 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Dès lors que nous utilisons Apache ou Nginx pour accéder à notre instance Wallabag, et non plus la commande ``php bin/console server:run --env=prod`` pour la démarrer, il faut prendre garde à octroyer les bons droits aux bons dossiers afin de préserver la sécurité de l'ensemble des fichiers fourni par le projet.
```
Comment (nicosomb) [inline docs/fr/user/installation.rst]: fourni > fournis

**Comment by nicosomb** [comment]
Thank you!

**Comment by foxmask** [comment]
Thanks you 2 4 wallabag ;)

**Comment by tcitworld** [comment]
What about the right access files part ?

**Comment by foxmask** [comment]
By default, they are good enough with the actual install peocess

**Pull request merged**